torch/onnx/experimental/fonnx/v1/custom_operators.py

```diff
@@ -0,0 +1,63 @@
+from contextlib import contextmanager
+
+
+def register_custom_operator(
+    domain: str,
+    operator: str,
+    opset_version: int,
+    symbolic_fn: Callable,
```

Comment:
nit: explicitly annotate with OnnxFunction. Although the `trace_only=True` option still yields us a Callable :(